client/lib/cart-values/index.js

```diff
@@ -253,7 +280,9 @@ export {
 };
 
 export default {
+	getQueryCoupon,
 	applyCoupon,
+	removeCoupon,
```

Comment:
Is there a strong reason to export these functions as part of an object in the default export?

Reply:
I understand why this export was added as a temporary measure. I'd like to discourage adding new names to the default export just because we're following the patterns already in place.

Reply:
well I'm not sure. I don't want to start making inconsistent exports. at least in these cases if someone updates `things.removeCoupon` it will still work without them needing to know about all this muck. I'd rather go around and fix the exports/imports in one sweep for each place that does it this way. I don't feel strongly about it but I'm nervous that by _not_ adding an export to the default object we'll be re-introducing new opportunities for runtime failure on static imports 😬

Reply:
Thanks for the feedback! I'm leaving both in the default export for now.